overrides/config/simplyjetpacks-client.cfg

```diff
@@ -29,7 +29,7 @@
     B:"Enable Fuel HUD"=true
 
     # When enabled, a HUD that displays the states (engine/mode/etc.) of the currently worn jetpack or flux pack will show.
-    B:"Enable State HUD"=true
+    B:"Enable State HUD"=false
```

Comment:
I think that keeping the HUD would be good, just for consistency and info (for new players). I like the other changes (such as the HUD removing while chatting).